src/mcode-pilot/components/TreatmentOptionsOutcomesIcons/TreatmentOptionsOutcomesIcons.jsx

```diff
@@ -0,0 +1,195 @@
+            this.setState({ displayedTreatment: null });
+        } else {
+            this.setState({ displayedTreatment: treatment });
+        }
+    }
+
+    renderInteractionHeader = () => {
+        const { selectedTreatment } = this.props;
+        return (
+            <div className="icons-interaction__header">
+                <div className="flex-3 flex-padding compare-header">
+                    {selectedTreatment ? 'comparing against' : 'compare'}
+                </div>
+
+                <div className="flex-1 flex-padding user-icon"><PersonIcon /></div>
+                <div className="flex-1 flex-padding overall-survival">Overall survival</div>
+            </div>
+        );
+    }
+
+    renderInteractionRow = (treatment, isSelectedTreatment = false) => {
+        if (!treatment) return null;
+        const { selectedTreatment, similarPatientTreatmentsData, timescaleToggle } = this.props;
+        const { displayedTreatment } = this.state;
+        const selectedTreatmentInData = selectedTreatment && this.getTreatmentIndex(selectedTreatment) !== -1;
+
+        const totalPatientsAllTreatments =
+            _.reduce(similarPatientTreatmentsData, (sum, treatment) => sum + treatment.totalPatients, 0);
+        const normalizedTotalPatients = Math.floor(treatment.totalPatients / totalPatientsAllTreatments * 100);
+        const numSurvive = this.getNumSurvive(treatment, timescaleToggle);
+        const isDisplayedTreatment = displayedTreatment && displayedTreatment.displayName === treatment.displayName;
+        const rowClass = `icons-interaction__row flex ${isSelectedTreatment ? 'selected-treatment' : ''} ${isDisplayedTreatment ? 'displayed-treatment' : ''}`;
+
+        let selectedNumSurvive;
+        let numSurviveDiff;
+        let numSurviveDiffClass;
+        if (selectedTreatmentInData) {
+            selectedNumSurvive = this.getNumSurvive(selectedTreatment, timescaleToggle);
+            numSurviveDiff = selectedNumSurvive - numSurvive;
+            numSurviveDiffClass =
+                `diff-num-selected ${numSurviveDiff < 0 ? 'positive' : ''} ${numSurviveDiff > 0 ? 'negative' : ''}`;
+        }
+
+        return (
+            <div className={rowClass} key={treatment.id} onClick={() => this.handleDisplayTreatment(treatment)}>
+                <div className="flex flex-3 flex-padding treatment-name">
+                    <div className="select-icon">
+                        {isSelectedTreatment
+                            ? <CompareSelectedIcon onClick={event => this.handleSelectTreatment(event, null)} />
+                            : <CompareUnselectedIcon onClick={event => this.handleSelectTreatment(event, treatment)} />
+                        }
+                    </div>
+
+                    <div className="display-name">{treatment.displayName}</div>
+                </div>
+
+                <div className="flex-1 flex-padding total-patients">({normalizedTotalPatients})</div>
```

Comment:
I think the point of the icons view is to represent the data out of 100 patients, which carries over to the data, so stating these are percentages is redundant. This could probably be made a little more clear, though. @dreeves4321 Thoughts?

Reply:
if this is the column with total number of patients, I agree, it should be clearer as # out of 100 patients. Maybe format the same way as the survival column (33/100)?

Reply:
Done!